lab5.py

```diff
@@ -0,0 +1,86 @@
+"""
+Module: lab5.py
+"""
+
+from typing import Union
+
+class File():
+    """
+    File class represents a file with certain characteristics.
+    """
+
+    def __init__(self, name, extension, size) -> None:
+        self.name = name
+        self.extension = extension
+        self.size = size
+
+    def get_info(self):
+        """
+        Get information about the file.
+        """
+        return f"File: {self.name}.{self.extension}, Size: {self.size} bytes"
+
+    def __str__(self):
+        return f"Your file: {self.name}.{self.extension}, Size: {self.size} bytes"
+
+    def __repr__(self):
+        return f"Your file: '{self.name}' '{self.extension}' {self.size} bytes"

```

Comment:
The "__repr__" method should ideally return a string that, when passed to "eval()", would recreate the object with the same state. In your case, it's not doing that. You might want to modify it to produce a string that can be used to recreate the object.

Reply:
Need to rewrite method "__repr__"